content/blog/build-embedded-swift-application-for-esp32c6/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Build Embedded Swift Application for ESP32-C6"
+date: 2024-06-21
+showAuthor: false
+authors:
+    - "juraj-michalek"
+tags: ["Embedded Systems", "ESP32-C6", "Espressif", "Swift"]
+---
+
+## Introduction
```

Comment:
I think it would be good to mention somewhere in the article that Embedded Swift is still **_experimental_** and under active development

Reply:
Good point. I've added **experimental** to several places.